contrib/init/systemd/README.md

```diff
@@ -3,20 +3,31 @@
 > NOTE: These instructions are valid for Ubuntu versions using `systemd` init
 > system.
 
-Run the following commands:
+## TL;DR
+1. Get the latest [release](https://github.com/Azure/azurefile-dockervolumedriver/releases)
+2. Put the binary into `/usr/bin/azurefile-dockervolumedriver`
+3. Get the .default and .service files and deploy them
+4. Reload systemd
+
+## In-depth walkthrough
 
 0. `sudo -s`
-0. Download the binary from “Releases” tab of the repo to `/opt/bin/`: `wget -qO/usr/bin/azurefile-dockervolumedriver [url]`
-0. Make it executable: `chmod +x /usr/bin/azurefile-dockervolumedriver`
+0. Use wget to get the `azurefile-dockervolumedriver.default` and `azurefile-dockervolumedriver.service` files from GitHub. These are in the `../contrib/init/systemd` directory.
+0. Download the binary from the "Releases" tab of the repo to `/usr/bin/azurefile-dockervolumedriver`
+    + Use wget to download to dir: `wget -qO/usr/bin/azurefile-dockervolumedriver https://github.com/Azure/azurefile-dockervolumedriver/releases/download/[CURRENT RELEASE NUMBER]/azurefile-dockervolumedriver`
+    + Make it executable `chmod +x /opt/bin/azurefile-dockervolumedriver`
 0. Save the `.default` file to `/etc/default/azurefile-dockervolumedriver`
+    + Copy the file: `cp azurefile-dockervolumedriver.default /etc/default/azurefile-dockervolumedriver`
```

Comment:
well, they don't have it on their disk at this point. so maybe we should point them to the file with something like:

    - download [azurefile-dockervolumedriver.default](../path/to/azurefile-dockervolumedriver.default) file first